lambda/prince/index.js

```diff
@@ -0,0 +1,85 @@
+const { execFile } = require('child_process');
```

Comment:
This file is heavily based on [an example](https://medium.com/@bruce_39084/setting-up-prince-on-aws-lambda-and-api-gateway-4d524dcb035b) from Prince. I'd like to rework this to accept input as JSON instead of a multipart form, which would eliminate about half of the code here.